code/extensions/Lumberjack.php

```diff
@@ -119,7 +119,7 @@ public function liveChildren($showAll = false, $onlyDeletedFromStage = false)
      */
     protected function getLumberjackTitle()
     {
-        if (method_exists($this->owner, 'getLumberjackTitle')) {
+        if ($this->owner->hasMethod('getLumberjackTitle')) {
```

Comment:
This would always return true as this extension is applied to the class. To override this method, you set it on the implementing DataObject directly.